code/controllers/subsystems/initialization/customitems.dm

```diff
@@ -6,42 +6,18 @@ SUBSYSTEM_DEF(customitems)
 	var/list/custom_icons_by_ckey = list()
 
 /datum/controller/subsystem/customitems/proc/get_json_paths_from_directory(var/category, var/directory)
-
 	. = list()
```

Comment:
Is this used? It used to return the loaded JSON, but now that's done elsewhere. Only noticed it because I realized the old version didn't have an explicit value returned. May be better to change the returns on line 12 and 16 to `return list()` if returning an empty list on failure is desirable—but no current code seems to rely on it returning an empty list rather than a null.